src/vs/platform/request/common/request.ts

```diff
@@ -155,6 +155,11 @@ function registerProxyConfigurations(scope: ConfigurationScope): void {
 				markdownDescription: localize('proxyKerberosServicePrincipal', "Overrides the principal service name for Kerberos authentication with the HTTP proxy. A default based on the proxy hostname is used when this is not set."),
 				restricted: true
 			},
+			'http.noProxy': {
+				type: 'string',
```

Comment:
Should this be an array? This will make the settings UI show a list like:
<img width="547" alt="image" src="https://github.com/microsoft/vscode/assets/9205389/db439f1d-a730-4a5b-b873-e4b27b9004e2">

Reply:
Hi @chrmarti , Thanks for the inputs. I have updated the PR to accept list of strings instead of a single string.